docs/get_started/getting_started_riscv_cmake.md

```diff
@@ -0,0 +1,148 @@
+# Getting Started on RISC-V with CMake
+
+<!--
+Notes to those updating this guide:
+
+    * This document should be __simple__ and cover essential items only.
+      Notes for optional components should go in separate files.
+-->
+
+This guide walks through cross-compiling IREE core runtime towards the RISC-V Linux platform. Cross-compiling IREE compilers towards RISC-V is not supported at the moment.
+
+Cross-compilation involves both a *host* platform and a *target* platform. One
+invokes compiler toolchains on the host platform to generate libraries and
+executables that can be run on the target platform.
+
+## Prerequisites
+
+### Set up host development environment
+
+The host platform should have been set up for developing IREE. Right now only
+Linux is supported. Please make sure you have followed the steps for
+[Linux](./getting_started_linux_cmake.md).
+
+### Install RISC-V Tools
+
+Execute the following script to download RISC-V toolchain and QEMU:
+```shell
+
+# In IREE source root
+$ ./build_tools/riscv/riscv_bootstrap.sh
+
+```
```

Comment:
Sorry I didn't phrase it clearly, I mean to add blank lines like this:

====START====
XXX
<----- this is a blank line ----->
```shell
# NO BLANK LINE
$ ./build_tools/riscv/riscv_bootstrap.sh
# NO BLANK LINE
```
<----- this is a blank line ----->
XXX
====END====

btw, no need for the comment.

Reply:
Got it. Sorry for the misunderstanding. I fixed that.